webpack.dev.js

```diff
@@ -39,7 +39,13 @@ module.exports = {
 			'commonjs2': 'react-dom/server',
 			'amd': 'react-dom/server',
 			'root': 'ReactDOMServer'
-		}
+		},
```

Comment:
Library was indeed authored with tabs. I dropped that battle eventually, but now is not the right time to diverge. This will be dealt with later in one shot.